docs/source/user/debugger.rst

```diff
@@ -38,6 +38,7 @@ Usage
 Here is a screencast to enable the debugger and set up breakpoints. The various steps are described more in depth below.
 
 .. image:: ./images/debugger/step.gif
+   :alt:
```

Comment:
A .gif file showing how to enable the debugger within the kernel.

Reply:
Hey there, thanks for contributing! If you'd be able to edit your above comment into a [GitHub suggestion](https://hackmd.io/@marslee/BksnYfWVK), I'd appreciate so I can make sure you get credit in the commit history. If not, I'll still add it with an extended description that mentions you.

Reply:
```suggestion
   :alt: A .gif file showing how to enable the debugger within the kernel.
```

Reply:
@isabela-pf My apologies for the slow response and thank you for fixing my mishap.

Reply:
No apology necessary! Thank *you* for writing alt text and improving our docs!